locales/en-US/snoozetabs.properties

```diff
@@ -0,0 +1,45 @@
+extDesc=An add-on to let you snooze your tabs for a while.
+popupTitle=Snooze tab until…
+contextMenuTitle=Snooze tab until…
+bookmarkFolderTitle=Snoozed Tabs
+notificationTitle=Tab woke up…
+
+confirmIconAltText=tab icon
+# LOCALIZATION NOTE: placeholder is the time the tab will snooze until, as per https://github.com/bwinton/SnoozeTabs/issues/60
+confirmTimeTitle=This tab will snooze until $TIME$.
```

Comment:
Per IRC discussion: if we could use a format localizers are used to, like `{time}`, it would probably reduce the risk of errors. 

See for example activity stream: https://github.com/mozilla/activity-stream-l10n/blob/master/l10n/templates/strings.properties#L35